src/sst/elements/mmu/simpleMMU.cc

```diff
@@ -129,8 +140,10 @@ void SimpleMMU::dup( unsigned fromPid, unsigned toPid ) {
 }
 
 void SimpleMMU::flushTlb( unsigned core, unsigned hwThread ) {
+#ifdef __SST_DEBUG_OUTPUT__
     m_dbg.debug(CALL_INFO_LONG,1,0,"core=%d hwThread=%d\n",core,hwThread);
 //    sendEvent( getLink(core,"itlb"), new TlbFlushEvent( hwThread ) );
```

Comment:
Is there a reason that you left this?

Reply:
Nope, I'll clean that up at some point...